.semaphore/semaphore.yml

```diff
@@ -91,14 +91,10 @@ blocks:
       commands:
       - cd api
     jobs:
-    - name: "make build"
+    - name: "make build / ci / static-checks"
```

Comment:
Not sure if this does much, but might be better to run one instance for slightly longer than it is to run three instances for shorter.

At a minimum it de-duplicates checking out the code and the other prologue steps which saves some CPU cycles.